src/telliot_feeds/reporters/tellor_flex.py

```diff
@@ -329,7 +330,7 @@ async def ensure_profitable(
 
         # Calculate profit
         rev_usd = tip / 1e18 * price_trb_usd
-        costs_usd = costs / 1e9 * price_matic_usd
+        costs_usd = costs / 1e9 * price_native_token
```

Comment:
just double checking if this is right, is it still nine decimal places?

Reply:
Yeah, gas price sources return recommended gas price in gwei, so need to convert to ether, then usd. I double checked each of the gas price source api docs